src/libraries/System.Reflection.Metadata/ref/System.Reflection.Metadata.cs

```diff
@@ -1256,9 +1252,7 @@ public static partial class ILOpCodeExtensions
     public partial class ImageFormatLimitationException : System.Exception
     {
         public ImageFormatLimitationException() { }
-#if !NETSTANDARD1_1
         protected ImageFormatLimitationException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
-#endif
         public ImageFormatLimitationException(string message) { }
         public ImageFormatLimitationException(string message, System.Exception innerException) { }
```

Comment:
Here and above message and innerExcception should be nullable

Reply:
Fixed